src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -1,4 +1,21 @@
 class Webui::CommentsController < Webui::WebuiController
+  before_action :require_login
+
+  def create
+    comment = Comment.new(permitted_params)
```

Comment:
And I think we should use the associations more, e.g. we could do:

```
User.current.comments << @commenter.comments.create(permitted_params)
```

And in permitted params you would only need ``body`` and ``parent_id``.

However, I don't like setting the ``parent_id`` without any validation ... but also can't think of a better solution right now ...